capstone-rs/src/arch/x86.rs

```diff
@@ -4,7 +4,7 @@ use core::convert::From;
 use core::{cmp, fmt, slice};
 
 use capstone_sys::{
-    cs_x86_op__bindgen_ty_1, x86_op_mem, x86_op_type, cs_x86, cs_x86_op};
+    cs_ac_type, cs_x86_op__bindgen_ty_1, x86_op_mem, x86_op_type, cs_x86, cs_x86_op};
```

Comment:
We may want to modify `capstone-sys` to treat `cs_ac_type` as a bitflag (which bindgen supports).